libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -804,6 +805,31 @@ class MapboxNavigation @VisibleForTesting internal constructor(
         )
     }
 
+    /**
+     * Requests road graph data update and invokes the callback on result.
+     * Use this method if the frequency of application relaunch is too low
+     * to always get the latest road graph data.
+     * Recreate [MapboxNavigation] instance in the callback when updates are available:
+     * ```
+     *   mapboxNavigation.requestRoadGraphDataUpdate(object : RoadGraphDataUpdateCallback {
+     *       override fun onRoadGraphDataUpdateInfoAvailable(
+     *           isUpdateAvailable: Boolean,
+     *           versionInfo: RoadGraphVersionInfo?
+     *       ) {
+     *           if (isUpdateAvailable) {
+     *               mapboxNavigation = MapboxNavigationProvider.create(...)
```

Comment:
```suggestion
     *               mapboxNavigation.onDestroy()
     *               mapboxNavigation = MapboxNavigationProvider.create(...)
```

Reply:
Oh, `MapboxNavigationProvider` under the hood so this isn't strictly needed. It would be good to at least mention that the old instance has to be destroyed.

Reply:
But the provider does the `onDestroy` invocation under the hood. Or do you mean that maybe the first one was created not via `MapboxNavigationProvider` so we should call `onDestroy` explicitly?

Reply:
I added the line, it definitely won't hurt.

Reply:
Alright, LGTM either way.